Reemit.Decompiler.Clr/Disassembler/OpcodeInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Reemit.Decompiler.Clr.Disassembler;
+
+// Todo:
+// User struct instead?
```

Comment:
`class`: 
<img width="536" alt="image" src="https://github.com/user-attachments/assets/6756dd16-7541-4d84-b513-e7d25dc29dde">

`readonly struct`:
<img width="507" alt="image" src="https://github.com/user-attachments/assets/582e03d5-7fa8-4f97-975b-b3eacc296173">

Benchmark code:
```csharp
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;
using Reemit.Decompiler.Clr.Disassembler;

var summary = BenchmarkRunner.Run<Bench>();

[MemoryDiagnoser]
public class Bench
{
    [Benchmark]
    public void DisasmMemBench()
    {
        byte[] methodBody = [114, 1, 0, 0, 112, 40, 13, 0, 0, 10, 42];
        using var stream = new MemoryStream(methodBody);
        var disasm = new StreamDisassembler(stream);
        var instr = disasm.Disassemble();
        var emitter = new InstructionEmitter();
        emitter.Emit(instr);
    }
}
```

We can make it even better by making `Instruction` a struct:
<img width="511" alt="image" src="https://github.com/user-attachments/assets/6c048a6e-da01-4c89-b92e-096d80028a7d">

Additional improvement can be achieved by also making `Operand` a struct:

<img width="519" alt="image" src="https://github.com/user-attachments/assets/5661cd91-1e1e-473a-a3c0-722af5160843">

Reply:
Awesome, good metrics. Changed them all.